packages/checklist/lib/pages/checklist_view.dart

```diff
@@ -1,21 +1,31 @@
 import 'dart:math';
 
-import 'package:digit_components/digit_components.dart';
+import 'package:auto_route/auto_route.dart';
+import 'package:checklist/checklist.dart';
+import 'package:checklist/utils/extensions/extensions.dart';
+import 'package:digit_components/blocs/location/location.dart';
```

Comment:
Why is the digit_components relative import not used?

Reply:
> Why is the digit_components relative import not used?

Since it is inside a package, we are doing a package import and not relative